code/modules/NTNet/relays.dm

```diff
@@ -1,3 +1,14 @@
+GLOBAL_LIST_EMPTY(ntnet_relays)
+
+///Checks whether NTNet is available by ensuring at least one relay exists and is operational.
+/proc/find_functional_ntnet_relay()
+	// Check all relays. If we have at least one working relay, ntos is up.
+	for(var/obj/machinery/ntnet_relay/relays as anything in GLOB.ntnet_relays)
+		if(!relays.is_operational)
+			continue
+		return TRUE
+	return FALSE
+
```

Comment:
how hot is this code? I guess ntnet_relays is a small list huh

Reply:
There's generally only 1-2 ntnet relays at any given time

Reply:
alright so redundant